examples/ssr-hydrate-axum-islands/.gitignore

```diff
@@ -0,0 +1,13 @@
+# Generated by Cargo
```

Comment:
The name of the directory I think that should be `ssr-islands-axum` because `hydrate` there can cause confusion, as the `hydrate` Leptos mode is not used in the example.